include/zephyr/dt-bindings/clock/stm32f410_clock.h

```diff
@@ -6,6 +6,9 @@
 #ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_STM32F410_CLOCK_H_
 #define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_STM32F410_CLOCK_H_
 
+/* CLK48MHz sources */
+#define STM32_SRC_CK48	(STM32_SRC_PLLI2S_R + 1)
+
```

Comment:
This should be added in stm32f4_clock.h. With this single define in a different file, there is a great risk that it would be overlooked if any other clock sources are added in the future (PLLSAI for example).
There is no risk to have that define even for SoC that don't have this clock (just like F410 that doesn't have a PLLI2S but has the associated define).